### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by amirnathani9**
# Console Logs
-added responsive image
-added rounded image
-added default input
-added error input

## GIF
input
image

## Preview Link
[Image Component Preview](https://deploy-preview-4--nomark-ui.netlify.app/components/image/image.html)
[Input Component Preview](https://deploy-preview-4--nomark-ui.netlify.app/components/input/input.html)

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    <!-- External Imports -->
+    <link rel="stylesheet" href="./image.css">
+    <link rel="stylesheet" href="../../styles/main.css">
+
+</head>
+
+<body>
+    <div class="section-center">
+        <!-- navbar -->
+        <nav class="navbar flex items-center justify-between pt-1 section-center z-5">
```
Comment (xii-vi) [inline components/image/image.html:21]: what is z-5?

**Review by xii-vi — APPROVED** [review]
_Just a single comment._ Resolve it to merge

File: components/image/image.html (hunk 1 above)
Comment (amirnathani9) [reply components/image/image.html:21]: z-index utility class

**Review by devdeadviz — APPROVED** [review]
Looks good to me!

**Review by viru2001 — COMMENTED** [review]
everything is good

**Pull request merged**